content/blog/2023/09/20/2023-09-20-Hacktoberfest-2023.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Hacktoberfest 2023"
+tags:
+- hacktoberfest
+- event
+- community
+- newcomer
+- outreach-programs
+author: jmMeessen
+links:
+  discourse: https://community.jenkins.io/t/hacktoberfest-2023/9613
+opengraph:
+  image: /images/hacktoberfest/hacktoberfest_2023_social_600x300.png
+---
+
+
+image:/images/hacktoberfest/hacktoberfest_2023_logo.png[Hacktoberfest, role=center]
+
+Once again, link:https://hacktoberfest.com[Hacktoberfest] is back!
+During this October event, everyone can celebrate and support open-source by contributing changes.
+
+This year's edition is special because it is the tenth time it takes place.
+The Jenkins community will participate once again in the event.
+We invite you to contribute to Jenkins projects but also, as maintainers, to welcome and help newcomers.
+
+The event is a good opportunity to cross the "imposter syndrome" threshold to contribute back to Open Source.
+A lot of helpful hands will be available to guide less experienced contributors in their first steps.
+It is also a great opportunity to acquire and exercise new skills (hard and soft) while improving your experience with CI/CD in general and Jenkins in particular.
+On top of this, it is a great way to "build the Jenkins dev muscle", should one want to compete to be part of the Jenkins Google Summer of Code 2024 adventure.
+
+September is the perfect time to prepare for Hacktoberfest.
+Get a jump start by finding projects to contribute to, adding "Hacktoberfest" tag to your projects, or familiarizing yourself with Git.
+
+Note that seen the increasing popularity of Hacktoberfest and to ensure its sustainability for another decade, the organization is 
+link:https://hacktoberfest.com/about/#digital-rewards[moving away from a free t-shirt reward to a digital reward].
+
+
+---
+
+== Contributors
+
+This is what **contributors** need to know to participate and complete Hacktoberfest:
+
+- Register anytime between September 26 and October 31 on the link:https://hacktoberfest.com/participation/[Hacktoberfest site].
+- Pull requests can be made in any link:https://github.com/orgs/jenkinsci/repositories?q=hacktoberfest[jenkinsci] or link:https://github.com/orgs/jenkins-infra/repositories?q=hacktoberfest[jenkins-infra] GitHub project that is participating in Hacktoberfest (look for the "hacktoberfest" topic)
+- Project maintainers must accept your pull requests for them to count toward your total
+- The new digital reward kit will include a customizable badge that evolves with each pull/merge request accepted by maintainers, representing the participant's journey in open-source and Hacktoberfest. 
+  Additionally, winners will receive unique badges featuring a delightful surprise and gifts from sponsors. 
+  To celebrate their success, participants will also be able to share their winner's badge on the Holopin Hacktoberfest Badge Board of Fame.
```

Comment:
```suggestion
  Additionally, participants will receive unique badges featuring a delightful surprise and gifts from sponsors. 
  Participants will also be able to proudly share their badge on the Holopin Hacktoberfest Badge Board of Fame.
```
updating to use participants instead of winners, as winners implies that there are losers of Hacktoberfest as well

Reply:
👍